src/admin/templates/xrootd/etc/cmsd-manager.cf.jinja

```diff
@@ -35,7 +35,7 @@ xrd.network nodnr
 xrd.network dyndns
 xrd.network cache 0
 
-all.manager {{ cmsd_manager }}:2131
+all.manager UNUSED:2131
```

Comment:
I was only trying to reduce unnecessary templatized parameters, as you and I have discussed as being a goal for simplicity. But, if it can be removed that would be nice. I think we must specify the port number for configuration, and in the docs `name:port` is given as the example. Perhaps I can remove the name, I'll try it.